api/envoy/extensions/filters/http/ip_tagging/v3/ip_tagging.proto

```diff
@@ -52,6 +69,23 @@ message IPTagging {
     repeated config.core.v3.CidrRange ip_list = 2;
   }
 
+  // Optional header to use for ip-tagging instead of using
+  // default header ``x-envoy-ip-tags``.
+  //
+  // This header will be sanitized based on the config in
+  // :ref:`ip_tag_header <envoy_v3_api_field_extensions.filters.http.ip_tagging.v3.IPTagging.ip_tag_header_action>`
+  // rather than the defaults for x-envoy prefixed headers.
+  string ip_tag_header = 5
+      [(validate.rules).string = {well_known_regex: HTTP_HEADER_NAME strict: false}];
+
+  // Control if the header in :ref:`ip_tag_header <envoy_v3_api_field_extensions.filters.http.ip_tagging.v3.IPTagging.ip_tag_header>`
+  // will be sanitized, or be appended to.
+  //
+  // This is ignored if :ref:`ip_tag_header <envoy_v3_api_field_extensions.filters.http.ip_tagging.v3.IPTagging.ip_tag_header>` is empty.
```

Comment:
As this field is closely related to the `ip_tag_header` field, it will be better to have them both under the same message type, and make the `ip_tag_header` field in that message type required (and probably move the `HeaderAction` to that inner message as well).